easybuild/easyconfigs/s/Seurat/Seurat-1.4.0-intel-2017a-R-3.4.0.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'RPackage'
+
+name = 'Seurat'
+version = '1.4.0'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'http://satijalab.org/seurat'
+description = "Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data."
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['https://github.com/satijalab/seurat/releases/download/v%(version_major_minor)s.0/']
+sources = ['%(name)s_%(version)s.tgz']
+
+dependencies = [('R', '3.4.0', '-X11-20170314')]
```

Comment:
Because it's not publised on CRAN, while all other extensions we install along with R are...